modules/sonarqube-service/main.tf

```diff
@@ -121,4 +121,3 @@ resource "aws_route53_record" "sonarqube" {
   ttl     = "300"
   records = [var.loadbalancer_dns_name]
 }
-
```

Comment:
Nitpicking but why remove this newline? All other terraform files end with this.

Reply:
I have no idea, it's not removed in my editor

Reply:
Fixed the new line